account/accounts.go

```diff
@@ -51,12 +51,13 @@ func NewManager(geth GethServiceProvider) *Manager {
 // BIP44-compatible keys are generated: CKD#1 is stored as account key, CKD#2 stored as sub-account root
 // Public key of CKD#1 is returned, with CKD#2 securely encoded into account key file (to be used for
 // sub-account derivations)
-func (m *Manager) CreateAccount(password string) (address, pubKey, mnemonic string, err error) {
+func (m *Manager) CreateAccount(password string) (*Info, string, error) {
```

Comment:
Up to you but returning a value instead of pointer should also work just fine. This is a very simple struct so pointer is probably not needed.

Reply:
good point, I updated the code